### PR TITLE
fix: guest account can't operate , log trace echart ui update

### PR DESCRIPTION
**Pull request by Qinyouzeng**

### CODE REVIEW

**Comment by doddgu** [comment]
@MayueCif PTAL

File: src/Services/Masa.Tsc.Service.Admin/Assets/I18n/en-US.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{  
+  "GuestAccountOperate": "Guest account is forbidden to operate"
```
Comment (MayueCif) [inline src/Services/Masa.Tsc.Service.Admin/Assets/I18n/en-US.json:2]: delete this and update stackmiddleware

File: src/Services/Masa.Tsc.Service.Admin/Domain/Topologies/Aggregates/TraceServiceRelation.cs [hunk 2]
```diff
@@ -1,16 +1,18 @@
 ﻿// Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
+using Nest;
```
Comment (MayueCif) [inline src/Services/Masa.Tsc.Service.Admin/Domain/Topologies/Aggregates/TraceServiceRelation.cs:4]: global using

File: src/Services/Masa.Tsc.Service.Admin/Domain/Topologies/Aggregates/TraceServiceState.cs [hunk 3]
```diff
@@ -1,32 +1,34 @@
 ﻿// Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
+using Nest;
```
Comment (MayueCif) [inline src/Services/Masa.Tsc.Service.Admin/Domain/Topologies/Aggregates/TraceServiceState.cs:4]: global using

File: src/Services/Masa.Tsc.Service.Admin/Assets/I18n/zh-CN.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{ 
+  "GuestAccountOperate": "演示账号禁止操作"
```
Comment (MayueCif) [inline src/Services/Masa.Tsc.Service.Admin/Assets/I18n/zh-CN.json:2]: others exceptions i18n missing

File: src/Web/Masa.Tsc.Web.Admin.Rcl/Pages/Dashboards/Configurations/Configuration.razor [hunk 5]
```diff
@@ -48,10 +48,15 @@
             <CascadingValue Value="ConfigurationRecord.IsEdit">
                 <CascadingValue Value="PanelGrids" IsFixed>
                     <CascadingValue Value="ConfigurationRecord">
+                        @*todo remove if*@
```
Comment (MayueCif) [inline src/Web/Masa.Tsc.Web.Admin.Rcl/Pages/Dashboards/Configurations/Configuration.razor:51]: specific instructions

**Review by MayueCif — COMMENTED** [review]
@Qinyouzeng

File: src/Services/Masa.Tsc.Service.Admin/Domain/Topologies/Aggregates/TraceServiceNode.cs [hunk 6]
```diff
@@ -1,20 +1,22 @@
 ﻿// Copyright (c) MASA Stack All rights reserved.
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
+using Nest;
```
[the remark this answers is not in the document]
Comment (Qinyouzeng) [reply src/Services/Masa.Tsc.Service.Admin/Domain/Topologies/Aggregates/TraceServiceNode.cs:4]: Nest 里面有个Query与Masa 的command 下的Query冲突，然后将Nest重命名会导致 Elasticseach使用的属性映射失效,如：[Keyword]

**Pull request merged**